### PR TITLE
ContactDetail / GroupDetail Header updates

### LINKED ISSUE #552: group name / avatars updates is buggy
steps to reproduce:
open a group chat profile, tap on edit, edit group name / icon , tap done
expectation:
image and name is updated in group detail view
actual behavior:
the old name/icon is still visible, going back to the chat and reopening the group chat profile (GroupDetailViewController) "fixes" the bug

### PR DESCRIPTION
**Pull request by nayooti**
fixes #552

### CODE REVIEW

File: deltachat-ios/ViewModel/ContactDetailViewModel.swift [hunk 1]
```diff
@@ -38,7 +39,10 @@ class ContactDetailViewModel: ContactDetailViewModelProtocol {
 
     var contactId: Int
 
-    var contact: DcContact
+    var contact: DcContact {
+        return DcContact(id: contactId)
```
Comment (cyBerta) [inline deltachat-ios/ViewModel/ContactDetailViewModel.swift]: nice :)

**Comment by cyBerta** [comment]
The issue still exists for images

**Comment by nayooti** [comment]
Added missing updates.

**Comment by nayooti** [comment]
tables now bounce as wished

**Pull request merged**